packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -109,7 +109,7 @@ export class TestAccount {
 		try {
 			const { birthtimeMs } = await fs.stat( this.getAuthCookiesPath() );
 			const nowMs = Date.now();
-			const threeDaysMs = 3 * 24 * 60 * 60 * 1000;
+			const threeDaysMs = 2 * 24 * 60 * 60 * 1000;
```

Comment:
We should probably change this variable name 🙂

Reply:
Doh, I completely blanked out on the name.

Reply:
Fixed in a0243051cd9a31869f7f324a5dc0100dcfea7d7d.